.idea/misc.xml

```diff
@@ -6,4 +6,4 @@
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```

Comment:
@dishantvyas15 Please revert these changes as we don't need them.

Reply:
@prayutsu, I have already made a commit (290ea32) to undo this change. Do I still need to do something else?

Reply:
EOF is missing, add one blank line at the end.

Reply:
@rt4914 Done.